sympy/assumptions/refine.py

```diff
@@ -1,12 +1,11 @@
 from __future__ import print_function, division
-
```

Comment:
IMO, deleting this line is relevant for the purpose of this PR. A separate code cleaning PR can be made for it. Thanks.